src/Translator/src/FileDumper/WebpackTranslationsDumper.php

```diff
@@ -30,8 +30,11 @@
  * @phpstan-type Locale string
  * @phpstan-type MessageId string
  */
-class TranslationsDumper
```

Comment:
This would be a massive BC break... not sure the "experimental" would be enough :)

Maybe should we keep it as alias / deprecated ? 

@kbond ?

Reply:
What about keeping the TranslationDumper as a wrapper that would call either or both the Webpack and AssetMapper dumper + the config dumper ?

Reply:
Why not ! That could reduce some code duplication too!